about/introduction.rst

```diff
@@ -1,14 +1,17 @@
 Introduction
 ============
 
-Our goal is to create a copylefted libre mobile operating system. One you can use, study, change and share; with all.
+Our goal is to create a copylefted libre mobile operating system. One you can use, study, change and share; with all of the code released under the GNU General Public license, V It's completely free, and anyone can inspect or modify the source code of any program that is included.
 
 About UBports
 -------------
 
-The project was founded by Marius Gripsgard in 2015 and in its infancy a place where developers could share ideas and educate each other in hopes of bringing the Ubuntu Touch platform to more mobile devices.
+Marius Gripsgard founded the project in 2015. In its infancy, developers could share ideas and educate each other to bring the Ubuntu Touch platform to more mobile devices. It now boasts a very active Linux community that brings along professional app developers to help improve the operating system for even more consumers.
+
+After Canonical suddenly announced plans to terminate support for Ubuntu Touch in April of 2017, UBports and its sister projects began work on the source code, maintaining and expanding its possibilities for the future. 
+
+Determined to preserve the Ubuntu touch, UBports and its sister projects began work on the kernel and other growing challenges. Today, UBports is a community of developers and enthusiasts that focuses on Research and Development within the Ubuntu source code.
```

Comment:
"the Ubuntu touch" -> "Ubuntu Touch"

"that focuses on Research and Development within the Ubuntu source code." -> I don't understand

why drop the foundation?

Reply:
Let's add this back !